src/masternode/sync.cpp

```diff
@@ -206,7 +206,7 @@ void CMasternodeSync::ProcessTick(const PeerManager& peerman)
                         // Now that the blockchain is synced request the mempool from the connected outbound nodes if possible
                         for (auto pNodeTmp : snap.Nodes()) {
                             bool fRequestedEarlier = m_netfulfilledman.HasFulfilledRequest(pNodeTmp->addr, "mempool-sync");
-                            if (pNodeTmp->nVersion >= 70216 && !pNodeTmp->IsInboundConn() && !fRequestedEarlier && !pNodeTmp->IsBlockRelayOnly()) {
+                            if (!pNodeTmp->IsInboundConn() && !fRequestedEarlier && !pNodeTmp->IsBlockRelayOnly()) {
```

Comment:
Nope, can't do this like this, we would need to bump minimum protocol version if you wanted, but generally we try to keep lots of support for protocol version. see https://github.com/dashpay/dash/blob/57fb0874ef180cd9aa8b7800a2a0842e2faba65e/src/version.h#L19-L20

@UdjinM6 please confirm my understanding

Reply:
We keep old proto support mostly for light/mobile clients. They do not have mempool so dropping this condition is ok-ish imo.

Reply:
Have bumped `MIN_PEER_PROTO_VERSION` to `70216` on the understanding that clients that don't have a mempool wouldn't care and nodes that do have a mempool would need to be on a far more recent version anyways.

Reply:
we should do this in it's own PR imo

Reply:
Change spun off into https://github.com/dashpay/dash/pull/6281